planning/behavior_velocity_planner/src/utilization/path_utilization.cpp

```diff
@@ -85,7 +87,8 @@ bool splineInterpolate(
   const std::vector<double> resampled_x = ::interpolation::slerp(base_s, base_x, resampled_s);
   const std::vector<double> resampled_y = ::interpolation::slerp(base_s, base_y, resampled_s);
   const std::vector<double> resampled_z = ::interpolation::slerp(base_s, base_z, resampled_s);
```

Comment:
@purewater0901 
I think z should be linear interpolated because it's linear interpolated in most of scene modules.

Reply:
done